src/shogun/distance/Distance.cpp

```diff
@@ -26,37 +26,10 @@
 #include <unistd.h>
 #endif
 
-#ifdef HAVE_PTHREAD
-#include <pthread.h>
-#endif
+#include <omp.h>
```

Comment:
but you dont need it because for finding out num available threads you should call `Parallel` singleton.